test/moonwall.config.json

```diff
@@ -65,6 +65,25 @@
                 ]
             }
         },
+        {
+            "name": "dev_simple_template",
```

Comment:
Do we want to run this in CI? My guess is no, because we almost never modify this template, and we already test the frontier template which should be very similar anyway.

Reply:
I would run it to be honest, in increases the coverage